pkg/noderesourcetopology/filter.go

```diff
@@ -89,16 +114,26 @@ func resourcesAvailableInAnyNUMANodes(logKey string, numaNodes NUMANodeList, res
 
 			resourceBitmask.Add(numaNode.NUMAID)
 			klog.V(6).InfoS("feasible", "logKey", logKey, "node", nodeName, "NUMA", numaNode.NUMAID, "resource", resource)
+
+			// according to TopologyManager, the preferred NUMA affinity, is the narrowest one.
+			// https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/topologymanager/policy.go#L155-L157
```

Comment:
let's use a link to a tag, so the link won't get stale if the code is refactored: https://github.com/kubernetes/kubernetes/blob/v1.24.0-rc.1/pkg/kubelet/cm/topologymanager/policy.go#L155